test_conformance/api/spirv_capability_deps.def

```diff
@@ -0,0 +1,312 @@
+//
```

Comment:
If the SPIRV-Headers become a hard dependency, maybe we should just generate this file at build time?

Reply:
Done.  I initially didn't want to add a build dependency on python, but we already need python to build SPIR-V binaries for the `spirv_new` test, so perhaps this is OK?

Maybe this test should move to the `spriv_new` executable instead of `api`, since it's SPIR-V related?  Then the dependency on python will all be in one place?

Or, if we decide we really don't want to bother with generation at build time, we can just drop the last commit.